cs/test/coreNS10/CoreNS10.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Common.Internal.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net45;</TargetFrameworks>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>UnitTest</RootNamespace>
+    <AssemblyName>Bond.UnitTestCoreNS10</AssemblyName>
+    <IsCodedUITest>False</IsCodedUITest>
+    <TestProjectType>UnitTest</TestProjectType>
+    <HasNonportableVersion>true</HasNonportableVersion>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <!-- This project is used to test the netstandard1.0 implementation of Bond -->
+    <BondTargetFramework>netstandard1.0</BondTargetFramework>
+    <BondIOTargetFramework>netstandard1.3</BondIOTargetFramework>
+    <!-- We need to specify the output directory since the .bond files are on another subdirectory -->
+    <BondOutputDirectory>$(IntermediateOutputPath)</BondOutputDirectory>
```

Comment:
Hmmm... Explicitly setting the `BondOutputDirectory` shouldn't be needed...

Reply:
If I remember correctly, it was using the codegen of the test/core/Core.csproj instead of using its own folder structure. I'll double check this.